mcs/build/tests.make

```diff
@@ -141,10 +143,49 @@ ifdef TEST_NUNITLITE_APP_CONFIG_RUNTIME
 	sed -i -e "/__INSERT_CUSTOM_APP_CONFIG_RUNTIME__/r $(TEST_NUNITLITE_APP_CONFIG_RUNTIME)" $(NUNITLITE_CONFIG_FILE)
 endif
 
+ifdef PLATFORM_AOT_SUFFIX
+
+DEDUP_DUMMY_CS=$(topdir)/class/lib/$(PROFILE)/DummyInflated.cs
```

Comment:
The architecture of dedup works by having all of the other (parallel) compilations skip emitting possibly duplicated methods and then by having one compilation (with no data dependency on the other ones) sweep over all of the source files that skipped methods and emit a single copy of each method into that AOT file.